app/src/main/java/com/teampophory/pophory/feature/album/AlbumListActivity.kt

```diff
@@ -0,0 +1,40 @@
+package com.teampophory.pophory.feature.album
+
+import android.os.Bundle
+import androidx.activity.viewModels
+import androidx.appcompat.app.AppCompatActivity
+import com.google.android.flexbox.AlignItems
+import com.google.android.flexbox.FlexDirection
+import com.google.android.flexbox.FlexWrap
+import com.google.android.flexbox.FlexboxLayoutManager
+import com.teampophory.pophory.databinding.ActivityAlbumListBinding
+import com.teampophory.pophory.feature.album.adapter.AlbumAdapter
+
+class AlbumListActivity : AppCompatActivity() {
+
+    private val albumAdapter = AlbumAdapter()
+    private val viewModel by viewModels<AlbumListViewModel>()
+    private val binding by lazy {
+        ActivityAlbumListBinding.inflate(layoutInflater)
+    }
```

Comment:
```suggestion
    private val binding by viewBinding(ActivityAlbumListBinding::inflate)
```

util쪽 함수 사용하는 방향으로 가볼까요?

Reply:
넹 좋습니다